packages/react-components/src/molecules/UserProfilePersonalText.tsx

```diff
@@ -126,6 +129,7 @@ const UserProfilePersonalText: FC<UserProfilePersonalTextProps> = ({
               <span css={{ color: tin.rgb }}>Add your location</span>
             )}
           </span>
+          {tags && <TagList tags={tags.map((tag) => tag.name)} />}
```

Comment:
It seems there's also the limit of 3 tags just like you added for `packages/react-components/src/organisms/SharedResearchCard.tsx`

<img width="492" alt="Screenshot 2024-03-25 at 14 01 03" src="https://github.com/yldio/asap-hub/assets/16595804/1e1224fb-a239-4b36-9b04-8d52dd47bb13">

also I think it's necessary to add a marginTop of 24

<img width="794" alt="Screenshot 2024-03-25 at 14 01 54" src="https://github.com/yldio/asap-hub/assets/16595804/1f2c6259-135a-4258-999a-3c5790b8c4a4">